### ISSUE TITLE
order_result ?

### ISSUE THREAD

**Issue by glebm** [sferik/mtgox#34]
order_result is mentioned in client.rb, but such method isn't present anywhere in the codebase. Is something amiss?

```
$ git grep order_result
lib/mtgox/client.rb:    # @return [String] order ID for the buy, can be inspected using order_result
lib/mtgox/client.rb:    # @return [String] order ID for the sell, can be inspected using order_result
lib/mtgox/client.rb:    # @return [String] order ID for the order, can be inspected using order_result
```

**Comment by davemarchevsky** [comment]
@sferik do you have any dangling local feature branches implementing this? Will implement in next few days otherwise. 

Actually, according to `git blame` I should be asking @demonbane

**Comment by sferik** [comment]
Yeah, I don't have any work in progress. I can't speak for @demonbane but I'm sure he wouldn't mind if you went ahead and implemented this behavior.

**Comment by demonbane** [comment]
I hit a roadblock with my PR (the v1 API appears to be incomplete), so feel free to implement. It won't screw me up any more than mtgox's incomplete API has already screwed me up. :crying_cat_face:

**Comment by sferik** [comment]
Do you know if this is fixed in API v2?

**Comment by demonbane** [comment]
Based on what I've been able to find, it looks like **none** of the APIs are actually _complete_. But how much of this is fact and how much of it is lack of documentation is hard to tell. V2 is the only API that has any documentation provided by MtGox, and that's in the form of a rather incomplete PDF. Both v0 and v1 just rely on community documentation, which appears to mostly be based on trial and error.

I believe that doing a v1/v2 hybrid would provide us a mostly complete implementation, but it would be a pain since v2 changes the way requests are authenticated.

The other option would be to have a v0/v1 hybrid which would be marginally less painful to implement, but it would technically be using two different deprecated APIs which is a whole different can of worms.

**Comment by sferik** [comment]
:disappointed:

**Comment by renich** [comment]
If you have any suggestions, make that at: https://github.com/MtGox/api

They're at it.
https://gist.github.com/christopherobin/611eff07a5d362de55ee
https://www.dropbox.com/s/z6iy5akoqwy1kbg/payment2013.png

**Comment by renich** [comment]
Maybe this helps: https://github.com/renich/mgc

At least it works with v2...

**Comment by jamescook** [comment]
It's in master now: https://github.com/sferik/mtgox/commit/40da764048aca09730190a7272623b51c140c541

**Comment by sferik** [comment]
Closed by #50.
